src/Core/src/Hosting/MauiAppBuilder.cs

```diff
@@ -160,7 +162,7 @@ public MauiApp Build()
 			{
 				foreach (var instance in initServices)
 				{
-					instance.Initialize(builtApplication.Services);
+					instance.Initialize(appScope.ServiceProvider);
```

Comment:
If we're passing a scoped service provider into `IMauiInitializeService`, what's the point of `IMauiInitializeScopedService`?

Reply:
Hmmm though it looks like this change now causes requests for the `IDispatcher` via `MauiApp.Services` to fail because it's not initialized on the right thread. 

So, I'll need to think on that for a bit or just go back to the path of having `MauiApp.Services` just be a scoped service.